docs/contributor/draft/rate-limit-api.md

```diff
@@ -0,0 +1,121 @@
+# Rate Limit Proposal
+
+## Spec 
+
+| field                                            | type        | description                                                     | required                      |
+|--------------------------------------------------|-------------|-----------------------------------------------------------------|-------------------------------|
+| wokloadSelector                                  | object      | in case of empty, it's applied for the whole namespace          | no                            |
```

Comment:
I think there was also proposed that the RateLimit CR will behave the same like EnvoyFilter, e.g. to apply to a namespace, or all namespaces, depending where it is applied. Information can be found in EnvoyFilter workloadSelector documentation: https://istio.io/latest/docs/reference/config/networking/envoy-filter/#EnvoyFilter

> If omitted, the set of patches in this configuration will be applied to all workload instances in the same namespace. If the EnvoyFilter is present in the config root namespace, it will be applied to all applicable workloads in any namespace.

Reply:
right, config root ns missed here